src/components/GithubIssues.jsx

```diff
@@ -2,30 +2,83 @@ import { connect } from 'react-redux';
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import * as githubActions from '../actions/githubActions';
-import TextComponent from './TextComponent';
 import LoadingComponent from './LoadingComponent';
+import RadarComponent from 'react-d3-radar';
 
-const renderData = projects =>
-  projects.map(project => <TextComponent text={project.name} key={project.name} size={project.issues.totalCount} />);
+let data = {
+  variables: [],
+  sets: [
+    {
+      key: 'Total Issues',
+      label: 'Total Issues',
+      values: {},
+    },
+    {
+      key: 'Open Issues',
+      label: 'Open Issues',
+      values: {},
+    },
+  ],
+};
+
+const renderData = projects => {
+  projects.totalIssues.map(project => {
+    data.variables.push({key: project.name, label: project.name});
+    data.sets[0].values[project.name] = project.issues.totalCount;
+    return data}
+  );
+  projects.openIssues.map(project => {
+    data.sets[1].values[project.name] = project.issues.totalCount;
+    return data}
+  );
+  return data;   
+}
 
 class GithubIssues extends React.Component {
   componentWillMount() {
-    this.props.github.fetchIssues();
+    this.props.github.fetchTotalIssues();
+    this.props.github.fetchOpenIssues();    
   }
 
   render() {
-    if (this.props.projects.length === 0) return <LoadingComponent/>;
+    if (!this.props.projects.openIssues || !this.props.projects.totalIssues) return <LoadingComponent/>;
     
-    return (
-      <div>
-        {
-          renderData(this.props.projects)
-        }
+    var radar = <div style={myStyle}>
+        <RadarComponent
+          width={600}
+          height={600}
+          padding={70}
+          domainMax={
+            Math.max.apply(
+              Math,this.props.projects.totalIssues.map(function(project) {
+                return project.issues.totalCount;
+              })
+            )
+          }
+          highlighted={null}
+          onHover={(point) => {
```

Comment:
Useful for debugging but we probably don't need to log these events if we are not using them at this point (onHover).